### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by ericgaspar**
- Update to 15.6.8
- Add README templates

### CODE REVIEW

**Comment by ericgaspar** [comment]
!testme

File: doc/DISCLAIMER.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+## YunoHost specific features
+
+* Disable third party website integration with Dropbox, Google, Trello, etc
```
Comment (alexAubin) [inline doc/DISCLAIMER.md:3]: It's not clear if this is meant as a "you should disable it yourself", or if the install version is the official diagramsnet but without these features ?

**Review by alexAubin — APPROVED** [review]
(LGTM apart from small clarification to be made in disclaimer)

**Comment by alexAubin** [comment]
(Bumpitybump, should be mergeable if we clarify disclaimer.md)

**Comment by ericgaspar** [comment]
!testme

**Comment by ericgaspar** [comment]
I changed the disclaimer to be less ambiguous.

cf. the remove calls to third party:
https://github.com/YunoHost-Apps/diagramsnet_ynh/blob/94ab670dcdb979adbde45b6830179b15ddf5f041/scripts/install#L80-L87

**Comment by alexAubin** [comment]
:heart:

**Pull request merged**